code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/events.cljs

```diff
@@ -854,6 +866,12 @@
      :fx [[:dispatch [::get-edges-for-edge-picker-modal]]
           [:dispatch [::table-plugin/reset-bulk-edit-selection [::spec/edge-picker-select]]]]}))
 
+(reg-event-fx
+  ::init-edge-picker-with-dynamic-filter
+  (fn [_ [_ fleet-filter]]
+    {:fx [[:dispatch [::set-edge-picker-fulltext-filter (utils/extract-fulltext-filter fleet-filter)]]
```

Comment:
Why not getting only what you need from the edges page filters?  Instead of trying to extract it from the query filter.

Reply:
This is in the edit deployment group page. It could be from a different session, and the filter is stored as a string server side.

Reply:
Support for fulltext search is adding too much code. And I don't see a requirement explicitly for it. Is there?

Reply:
Indeed. And when I try to put it as part of the additional filter I get this error on `PUT /nuvlaboxes`:
```
{
    "status": 500,
    "message": "unexpected exception querying: {:root_cause [{:type \"illegal_argument_exception\", :reason \"Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [fulltext] in order to load field data by uninverting the inverted index. Note that this can use significant memory.\"}], :type \"search_phase_execution_exception\", :reason \"all shards failed\", :phase \"query\", :grouped true, :failed_shards [{:shard 0, :index \"nuvla-nuvlabox\", :node \"u4Bh9OVCRuuGLEm7Vgx32w\", :reason {:type \"illegal_argument_exception\", :reason \"Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [fulltext] in order to load field data by uninverting the inverted index. Note that this can use significant memory.\"}}], :caused_by {:type \"illegal_argument_exception\", :reason \"Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [fulltext] in order to load field data by uninverting the inverted index. Note that this can use significant memory.\", :caused_by {:type \"illegal_argument_exception\", :reason \"Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [fulltext] in order to load field data by uninverting the inverted index. Note that this can use significant memory.\"}}}"
}
```
I think I will just enable dynamic filters on additional filters on this iteration. We can come back to this later if there is a request.